tools/summaries/fetch_place_summaries.py

```diff
@@ -237,12 +230,19 @@ def build_template_summaries_for_sitemap(
 @click.option('--batch_size',
               default=_BATCH_SIZE,
               help='how many places to process at once')
-def main(sitemap: str, stat_var_json: str, output_file: str, batch_size: int):
+@click.option('--num_skip_lines',
```

Comment:
nit: num_skip_lines was slightly confusing to me at first (in terms of naming), maybe something like 'sitemap_start_idx'

Reply:
Good idea, renamed to "start index" and updated the help line and comments to clarify.